lib/fastlane/plugin/revenuecat/helper/revenuecat_helper.rb

```diff
@@ -0,0 +1,156 @@
+require 'fastlane_core/ui/ui'
+require 'fastlane/action'
+
+module Fastlane
+  UI = FastlaneCore::UI unless Fastlane.const_defined?(:UI)
+
+  module Helper
+    class RevenuecatHelper
+      def self.replace_version_number(previous_version_number, new_version_number, files_to_update, files_to_update_without_prerelease_modifiers)
+        previous_version_number_without_prerelease_modifiers = previous_version_number.split("-")[0]
+        new_version_number_without_prerelease_modifiers = new_version_number.split("-")[0]
+
+        files_to_update.each do |file_to_update|
+          replace_in(previous_version_number, new_version_number, file_to_update)
+        end
+        files_to_update_without_prerelease_modifiers.each do |file_to_update|
+          replace_in(previous_version_number_without_prerelease_modifiers, new_version_number_without_prerelease_modifiers, file_to_update)
+        end
+      end
+
+      def self.auto_generate_changelog(repo_name, old_version, github_token, rate_limit_sleep, verbose)
+        last_version = Actions.sh("git describe --tags --abbrev=0").strip
+        old_version ||= last_version
+        UI.important("Auto-generating changelog since #{old_version}")
+
+        org = "RevenueCat"
+
+        path = "/repos/#{org}/#{repo_name}/compare/#{old_version}...HEAD"
+
+        # Get all commits from previous version (tag) to HEAD
+        resp = github_api(path: path, api_token: github_token)
```

Comment:
Hey @joshdholtz! I was running into a new issue, in case you have any suggestions about this :sweat_smile:. I'm trying to move the logic to auto generate a changelog based on the commits since the last release to the new plugin. This code currently uses the `github_api` action, as seen here. 

I moved this code out of the action itself and into the helper, to keep the action cleaner. When I try to call `github_api` directly like this (code not working currently), it can't find the action. It does find it if I do `Actions::GithubApiAction.run`, but doing it that way, I don't get the input validation, nor the default params on that action. I tried using `other_action.github_api`, but it seems it can't find `other_action` in the helper (tried a few things without luck).

Should I move this logic back to the `action` so I can use `other_action`, or do you know a better way to call other actions from the helper?